tools/postgres/schema.sql

```diff
@@ -34,6 +34,7 @@ CREATE TABLE webknossos.annotations(
   _task CHAR(24),
   _team CHAR(24) NOT NULL,
   _user CHAR(24) NOT NULL,
+  _publication CHAR(24),
```

Comment:
there is a foreign key section at the bottom of schema.sql. this reference should be added there (also to the evolution)